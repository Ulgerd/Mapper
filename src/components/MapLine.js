import React from 'react';
import { Polyline } from 'react-yandex-maps';

export function MapLine(props) {
  return (
    <Polyline
      geometry={props.coords}
      options={{
        strokeColor: "#000",
        strokeWidth: 4,
        strokeOpacity: 0.5
      }}
    />
  );
}

export default MapLine;
