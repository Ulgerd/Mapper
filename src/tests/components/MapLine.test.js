import React from 'react';
import { MapLine } from '../../components/MapLine.js';
import renderer from 'react-test-renderer'

import {YMaps, Map, Polyline } from 'react-yandex-maps';

// null =(

describe('MapLine component tests',()=>{
    it('renders correctly', () => {
      let cat = renderer
        .create(
          <YMaps>
          <Map           width={'40em'}
                    height={'40em'}
                    state={{center: [55.75, 37.57], zoom: 9}}>
          <MapLine/>
          </Map>
</YMaps>
        )
        .toJSON();
      expect(cat).toMatchSnapshot();
    });
});
