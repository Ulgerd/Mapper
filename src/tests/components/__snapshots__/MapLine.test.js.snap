// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapLine component tests renders correctly 1`] = `null`;
